stage0/registration.go

```diff
@@ -31,6 +31,7 @@ import (
 	"github.com/appc/spec/schema"
 	"github.com/appc/spec/schema/types"
 	"github.com/hashicorp/errwrap"
+	"github.com/prometheus/common/log"
```

Comment:
Old versions of goimports add spurious imports even if there's a package-level variable defined.

As good a reason as any to update!